src/scenario/message_media_text_blob.h

```diff
@@ -0,0 +1,41 @@
+#ifndef MESSAGE_MEDIA_TEXT_BLOB_H
+#define MESSAGE_MEDIA_TEXT_BLOB_H
+
+#include "core/buffer.h"
+
+#define MESSAGE_MEDIA_TEXT_BLOB_MAXIMUM_SIZE (1 * 1024 * 1024)
```

Comment:
Should this rather be done with dynamic memory assignment?
Same for line just below.

(I initially went that route but ran into problems. Went with static values to simplify while I got the code working. Willing to try to refactor to dynamic if that would be better.)

Reply:
Sidenote. Yesterday I tried doing dynamic memory in a separate branch. Seemed to work fine during testing (saving, loading, etc.) except that an icon on the UI had a graphical glitch, indicating there is some hidden memory leak or something far more serious going on.

Reply:
I know this isn't the main PR anymore, but I can try helping you out with this if you need.